modules/utils/claws/utils/box.hpp

```diff
@@ -5,6 +5,13 @@
 
 namespace claws
 {
+  template<typename T>
+  inline constexpr bool is_boxable_v = not std::is_class_v<T>;
```

Comment:
(chuckling at `not`)

Reply:
I felt like the `not` here was nice since we have an almost-pure-english equation :p

The `inline` can actually seem quite odd, but IIRC, because of a quirk in the standard, the linkage of specializations for variable templates is not really explicitly specified (if you're interested, I think that's one of the opened issues on open-std).
Specifying inline here ensures we actually get the desired effect.

Reply:
I'm ok with it, but it still seems odd, especially since IIRC constexpr implies inline explictly.

Reply:
Ok, cppref says: "A constexpr specifier used in a function or static member variable (since C++17) declaration implies inline."

Reply:
We'll leave it in case.